src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -114,24 +114,24 @@ private BoundPattern BindDiscardPattern(DiscardPatternSyntax node, TypeSymbol in
                 hasErrors = true;
             }
 
-            return BindConstantPattern(node, innerExpression, inputType, node.Expression, hasErrors, diagnostics, out _);
+            return BindConstantPattern(node, innerExpression, operandType, node.Expression, hasErrors, diagnostics, out _);
         }
 
         internal BoundConstantPattern BindConstantPattern(
             CSharpSyntaxNode node,
-            TypeSymbol inputType,
```

Comment:
>private [](start = 8, length = 7)

`static`?

Reply:
I cannot tell what this comment refers to, but there are no methods near here that could be static.

---
In reply to: [180579394](https://github.com/dotnet/roslyn/pull/25963#discussion_r180579394) [](ancestors = 180579394)

Reply:
The comment referred to `CheckIsTupleElement`.

---
In reply to: [180602793](https://github.com/dotnet/roslyn/pull/25963#discussion_r180602793) [](ancestors = 180602793,180579394)